MemoryManagement/src/memorymanagementAlgorithm/Segment.java

```diff
@@ -1,5 +1,14 @@
 package memorymanagementAlgorithm;
 
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+/**
+ *
+ * @author ahmed
+ */
 public class Segment {
 
     private static int SEGMENT_ID = 0;

```

Comment:
there must be a way to reset the ID every process. So the segments of the process always start with ID 0.

Reply:
not fixed
every segment got it elite  id 
not need to give it a zero .